langgraph_bot/nodes/load_data_node.py

```
@@ -12,13 +12,15 @@ def load_data(state:State):
     # FOR PRODUCTHUNT API ONLY.
     node = {
-        "id": "1042085",
-        "name": "Qoder JetBrains Plugin",
-        "tagline": "Understands your backend's real complexity, not just syntax",
-        "createdAt": "2025-11-28T08:01:00Z",
-        "votesCount": 382,
-        "description": "AI plugin for JetBrains that understands backend projects architecturally. Accesses Spring Bean graphs, database schemas, and framework semantics directly — not just superficial code. Provides context-aware suggestions for complex systems with 100K+ files.",
-        "reviewsCount": 6,
+        "id": "1044782",
+        "name": "8bitcn/ui",
+        "tagline": "8‑bit UI components that work in any framework",
```

Ruff (0.14.10)

String contains ambiguous `‑` (NON-BREAKING HYPHEN). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)